kgPanelsConfig/PanelHelper.lua

```diff
@@ -1274,7 +1274,7 @@ function kgPanelsConfig:CreatePanelMenu(panelName, panelData, isDefault)
 							xOffset = {
 								type = "input",
 								name = L["X Offset"],
-								desc = L["X offset from center. NOTE: positive for up, negative for down."],
+								desc = L["X offset from center. NOTE: postive for up, negative for down."],
```

Comment:
These localisation keys were spell-corrected but in the actual localisation tables the key still has the spelling misstake and therefore it cannot match the keys. The actual value will still be correct though.

Reply:
From `kgPanels-enUS.lua`
`L["X offset from center. NOTE: postive for up, negative for down."] = "X offset from center. NOTE: Positive for up, negative for down."`